packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -65,3 +70,22 @@
 .edit-site-screen-background-color__control {
 	padding: $grid-unit-20;
 }
+
+.edit-site-global-styles-variations_item {
+	box-sizing: border-box;
+	padding: 3px;
+	border-radius: $radius-block-ui;
+	border: $gray-200 1px solid;
+
+	&.is-active {
+		border: $gray-900 1px solid;
```

Comment:
As referred above I guess for borders instead of 1px we can use variable $border-width;